signatures/alert_signatures.py

```diff
@@ -0,0 +1,1012 @@
+    "ET P2P BitTorrent DHT nodes reply": MicroAttackStage.DATA_DELIVERY,
+    "ET P2P BitTorrent DHT announce_peers request": MicroAttackStage.DATA_DELIVERY,
+    "ET POLICY IP Check Domain (whatismyip in HTTP Host)": MicroAttackStage.INFO_DISC,
+    "ET POLICY TeamViewer Dyngate User-Agent": MicroAttackStage.INFO_DISC,
+    "ET SCAN Non-Allowed Host Tried to Connect to MySQL Server": MicroAttackStage.INFO_DISC,
+    "ET POLICY User-Agent (Launcher)": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET POLICY PDF With Embedded File": MicroAttackStage.DATA_DELIVERY,
+    "ET P2P Bittorrent P2P Client User-Agent (Transmission/1.x)": MicroAttackStage.DATA_DELIVERY,
+    "ET WEB_CLIENT Hex Obfuscation of String.fromCharCode % Encoding": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET WEB_CLIENT Hex Obfuscation of charCodeAt % Encoding": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET WEB_CLIENT Hex Obfuscation of document.write % Encoding": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET WEB_CLIENT Hex Obfuscation of Script Tag % Encoding": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET WEB_CLIENT Hex Obfuscation of unescape % Encoding": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET WEB_CLIENT Hex Obfuscation of substr % Encoding": MicroAttackStage.ARBITRARY_CODE_EXE,
+    "ET POLICY DNS Query For XXX Adult Site Top Level Domain": MicroAttackStage.NON_MALICIOUS,
+    "ET POLICY Cleartext WordPress Login": MicroAttackStage.PRIV_ESC,
+    "ET POLICY Http Client Body contains passwd= in cleartext": MicroAttackStage.PRIV_ESC,
+    "ET POLICY Http Client Body contains pass= in cleartext": MicroAttackStage.PRIV_ESC,
+    "ET POLICY Http Client Body contains pwd= in cleartext": MicroAttackStage.PRIV_ESC,
+    "ET POLICY curl User-Agent Outbound": MicroAttackStage.DATA_DELIVERY,
+    "ET POLICY libwww-perl User-Agent": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET POLICY Python-urllib/ Suspicious User Agent": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET SCAN Nessus FTP Scan detected (ftp_anonymous.nasl)": MicroAttackStage.INFO_DISC,
+    "ET POLICY GNU/Linux APT User-Agent Outbound likely related to package management": MicroAttackStage.NON_MALICIOUS,
+    "ET POLICY GNU/Linux YUM User-Agent Outbound likely related to package management": MicroAttackStage.NON_MALICIOUS,
+    "ET TROJAN Suspicious User-Agent (WindowsNT) With No Separating Space": MicroAttackStage.COMMAND_AND_CONTROL,
+    "ET POLICY HTTP traffic on port 443 (POST)": MicroAttackStage.NON_MALICIOUS,
+    "ET POLICY Vulnerable Java Version 1.7.x Detected": MicroAttackStage.NON_MALICIOUS,
+    "ET POLICY Outdated Flash Version M1": MicroAttackStage.NON_MALICIOUS,
+    "ET POLICY OpenVPN Update Check": MicroAttackStage.NON_MALICIOUS,
+    "ET POLICY DynDNS CheckIp External IP Address Server Response": MicroAttackStage.INFO_DISC,
+    # "ET POLICY DNS Query for TOR Hidden Domain .onion Accessible Via TOR": MicroAttackStage.DEFENSE_EVASION,
```

Comment:
What should I do with these commented alert signatures? Keep them, or can they be removed?

Reply:
Keep them commented, might be useful some day